lib/tasks/rake_arguments.rb

```diff
@@ -0,0 +1,5 @@
+module RakeArguments
+  def self.flags
+    ARGV.drop_while { |e| e != '--' }.drop(1)
```

Comment:
It's not a method it's a variable assignment.

Just delete the tests, they're not necessary any more.